models/classes/runner/time/TimerStrategyService.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA ;
+ *
+ */
+
+namespace oat\taoQtiTest\models\runner\time;
+
+use oat\oatbox\service\ConfigurableService;
+
+class TimerStrategyService extends ConfigurableService implements TimerStrategyInterface
+{
+    const SERVICE_ID = 'taoQtiTest/TimerStrategyService';
```

Comment:
service_id better to move into the TimerStrategyInterface

Reply:
how do you think: do we need this service? can we use it without TimerService? outside of this service, probably it would be better to delete SERVICE_ID from this class and initialize it like QtiTimerService(['timer_strategy' => new TimerStrategyService()]) (it's not working code, but you have an idea)

Reply:
We use it in OdsResultsPayload to generate json and in DeliveryExecutionManagerService